Changes.md

```diff
@@ -25,6 +25,7 @@ API
 Fixes
 -----
 - Node UI : "userDefault" metadata now evaluated with the script context, so it can depend on script variables.
+- ImageStats : Results for min/max now correctly reflect zero values outside the data window.
```

Comment:
While this is definitely a bug fix, it does also change the output of the node in a way that might not suit every situation.